### DISCUSSION TITLE
osu! defaulting to lowest refresh rate out of two monitors

### DISCUSSION THREAD

**Discussion by NBBZ** [ppy/osu#14791] [Q&A] [answered]
I already know what causes this issue, my second monitor is 60 Hz an osu! is defaulting to the lowest refresh rate it finds instead of 144 Hz, when I unplug the second monitor it works in 144 Hz, I am using artix linux and I have seen similar issues but I couldn't figure out the solution, or if there even is one.
wayland is not an option yet, sorry.

**Comment by bdach** [comment, marked as answer]
likely duplicate of https://github.com/ppy/osu/issues/8963

**Reply by NBBZ** [reply]
yes you are correct, i will leave a ditailed comment there, can you close this discossion then? (i am not sure how to do that)
